src/components/PoolOverview.tsx

```diff
@@ -35,6 +35,15 @@ function PoolOverview({
           }
         : acc
     }, {} as Partial<Record<Partners, string>>),
+    apy: poolData.apy
```

Comment:
generally prefer es6 string templates over concatenating strings. Also `.toFixed` returns a string so `String()` is unnecessary 
```
`${parseFloat(poolData.apy * 100).toFixed(4)}%`
```
0.123456 -> "12.3456%"